test/nlpmodels.jl

```diff
@@ -33,7 +36,7 @@ Please look at the warnings displayed at the end.
 jac_inconsistencies = []
 hess_inconsistencies = []
 
-@testset "$name" for name in problem_names()
+@testset "$name" for name in Symbol.(OptimizationProblems.meta[!, :name])
```

Comment:
Maybe `problem_names` could be exported from `SparseConnectivityTracerBenchmarks` as well?